packages/client/src/features/accessibility/move-zoom/move-handler.ts

```diff
@@ -0,0 +1,198 @@
+        return Action.hasKind(object, KIND);
+    }
+
+    export function create(moveX: number, moveY: number): MoveViewportAction {
+        return { kind: KIND, moveX, moveY };
+    }
+}
+
+/**
+ * Action for triggering moving of elements.
+ */
+export interface MoveElementAction extends Action {
+    kind: typeof MoveElementAction.KIND;
+    /**
+     * used to specify the elements to be zoomed in/out
+     */
+    elementIds: string[];
+    /**
+     * used to specify the amount to be moved in the x-axis
+     */
+    moveX: number;
+    /**
+     * used to specify the amount to be moved in the y-axis
+     */
+    moveY: number;
+}
+
+export namespace MoveElementAction {
+    export const KIND = 'moveElementAction';
+
+    export function is(object: any): object is MoveElementAction {
+        return Action.hasKind(object, KIND);
+    }
+
+    export function create(elementIds: string[], moveX: number, moveY: number): MoveElementAction {
+        return { kind: KIND, elementIds, moveX, moveY };
+    }
+}
+
+/* The MoveViewportHandler class is an implementation of the IActionHandler interface that handles
+moving of the viewport. */
+@injectable()
+export class MoveViewportHandler implements IActionHandler {
+    @inject(EditorContextService)
+    protected editorContextService: EditorContextService;
+
+    @inject(TYPES.IActionDispatcher) protected dispatcher: IActionDispatcher;
+    protected readonly throttledHandleViewportMove = throttle((action: MoveViewportAction) => this.handleMoveViewport(action), 150);
+
+    handle(action: Action): void | Action | ICommand {
+        if (MoveViewportAction.is(action)) {
+            this.throttledHandleViewportMove(action);
+        }
+    }
+
+    handleMoveViewport(action: MoveViewportAction): void {
+        const viewport = findParentByFeature(this.editorContextService.modelRoot, isViewport);
+        if (!viewport) {
+            return;
+        }
+        this.dispatcher.dispatch(this.moveViewport(viewport, action.moveX, action.moveY));
+    }
+
+    protected moveViewport(viewport: SModelRoot & Viewport, offsetX: number, offSetY: number): SetViewportAction {
+        const newViewport: Viewport = {
+            scroll: {
+                x: viewport.scroll.x + offsetX,
+                y: viewport.scroll.y + offSetY
+            },
+            zoom: viewport.zoom
+        };
+
+        return SetViewportAction.create(viewport.id, newViewport, { animate: true });
+    }
+}
+
+/* The MoveElementHandler class is an implementation of the IActionHandler interface that handles
+moving elements. */
+@injectable()
+export class MoveElementHandler implements IActionHandler {
+    @inject(EditorContextService)
+    protected editorContextService: EditorContextService;
+    @inject(TYPES.IActionDispatcher) protected dispatcher: IActionDispatcher;
+    protected readonly throttledHandleElementMove = throttle((action: MoveElementAction) => this.handleMoveElement(action), 500);
```

Comment:
The timeouts in `move-handler.ts` and `zoom-handler.ts` differ in the different handlers. Could you please align them, thanks!